src/child/containers/sidebars/Sidebar.js

```diff
@@ -166,7 +172,8 @@ Sidebar.propTypes = {
     sidebar: PropTypes.object.isRequired,
     windowState: windowStateShape.isRequired,
     selection: selectionShape.isRequired,
-    favourites: favouritesShape.isRequired
+    favourites: favouritesShape.isRequired,
+    numberOfClosedWindows: PropTypes.number.isRequired
```

Comment:
I'd be tempted to name as `closedWindowsCount` as it's more concise but just as descriptive